m4_sessions/1705-inning/group_six.md

```diff
@@ -10,5 +10,13 @@
 
 ### Outline: 
 
-<p>10 minutes for answering questions. 15 minutes whiteboard process. 25 minutes group discussions. 10 minutes for conclusion. </p>
+  * Answer question on their own (10 minutes):
```

Comment:
I think this will be a little too long--what if you have them write for 5 minutes, share with a partner for 3 minutes, and then hear from the group (maybe make a list on the board with different ideas)